server/lib/enrich/article.js

```diff
@@ -33,6 +33,14 @@ module.exports = exports = function article(content, format) {
 
 	content.hasGraphics = Boolean(content.contentStats && content.contentStats.graphics);
 
+	// Currently embeds can have more than one item for each picture - for different screen sizes
+	// We are assuming that canBeSyndicated is the same for all sizes of a picture
+	// We are asking if ALL Graphics can be syndicated, which means as long as at least one item
+	// can't be, the answer to this is 'no'
+	const atLeastOneGraphicCantBeShared = content.embeds && content.embeds.filter(embed => embed && embed.type.endsWith('Graphic')).some(item => item.canBeSyndicated !== 'yes');
+
+	content.canAllGraphicsBeSyndicated = Boolean(atLeastOneGraphicCantBeShared);
```

Comment:
```suggestion
	content.canAllGraphicsBeSyndicated = !atLeastOneGraphicCantBeShared;
```
If at least one graphic can't be syndicated, then all graphics can't be syndicated.

Likewise, if there is no graphics that can't be syndicated, then all graphics can be syndicated.

Reply:
My bad. Got confused with double negatives 😅